app/src/main/java/org/projectbuendia/client/ui/OdkActivityLauncher.java

```diff
@@ -102,6 +132,11 @@ public static void fetchAndShowXform(
         final int requestCode,
         @Nullable final org.odk.collect.android.model.Patient patient,
         @Nullable final Preset fields) {
+        LOG.i("Trying to fetch it from cache.");
```

Comment:
@capnfabs when is an app-wide sync triggered?

If we have 10,000 patients, would it be unnecessarily slow to do an app-wide sync just to get updated forms? Or not, now that we have incremental sync?